backend/molgenis-emx2-webapi/src/main/java/org/molgenis/emx2/web/ZipApi.java

```diff
@@ -174,21 +174,35 @@ static void getZippedReports(Context ctx) throws IOException {
     }
   }
 
-  static void generateReportsToStore(Context ctx, TableStore store) throws JsonProcessingException {
+  static void generateReportsToStore(
+      @org.jetbrains.annotations.NotNull Context ctx, TableStore store)
```

Comment:
Why this not null annotation?

Reply:
ah, I think I added that by accident. Is a way to prevent null exceptions but didn't mean to do this.